src/common/select-objects.js

```diff
@@ -57,7 +57,10 @@ const getLabel = object =>
   predicate: propTypes.func,
   required: propTypes.bool,
   value: propTypes.any,
-  xoContainers: propTypes.array,
+  xoContainers: propTypes.oneOfType([
+    propTypes.array,
+    propTypes.object
```

Comment:
Yes, because containers should be sorted, if you simply `sort()` them in your selectors creator it should be fine.